src/HOCON/Impl/Utils.cs

```diff
@@ -182,5 +193,19 @@ public static bool NeedQuotes(this string s)
         public static bool NeedTripleQuotes(this string s)
             => s.NeedQuotes() && s.Contains(Utils.NewLine);
 
+        public static string AddQuotesIfRequired(this string s)
```

Comment:
You can use ToHoconSafe() string extension instead.

Reply:
You are right. I was concerned that a key may contain the `\` character. Would that be a problem?

Reply:
It shouldn't be a problem, unless you can found a case where there is a problem? If there is a problem, then ToHoconSafe() should be fixed too.